### PR TITLE
python: conversion to H5 from pocolog/rock logs

### PR DESCRIPTION
**Pull request by jhidalgocarrio**
This PR gives the possibility to convert [Rock](https://www.rock-robotics.org/) logs to H5 format. Afterward, the e2calib process stays the same.

### CODE REVIEW

**Comment by jhidalgocarrio** [comment]
@Manasi94 could you please check the current PR

File: python/pocolog_convert.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import argparse
```
Comment (magehrig) [inline python/pocolog_convert.py:1]: Why do we need this file?
this is more or less a duplicate of convert.py

File: python/data/provider.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def get_t_end_us(self):
     def get_events_until(self, time: int) -> Optional[Events]:
         # Returns events if successful, None otherwise.
 
-        assert time > self.last_time, f'time = {time}, last_time={self.last_time}'
+        assert time >= self.last_time, f'time = {time}, last_time={self.last_time}'
```
Comment (magehrig) [inline python/data/provider.py:127]: why the change?

File: python/conversion/pocolog.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+# Pocolog bindings
+from pocolog_pybind import *
+
+import tqdm
+import warnings
+import numpy as np
+
+from data.format import Events
+
+class EventAccumulator:
```
Comment (magehrig) [inline python/conversion/pocolog.py:12]: This class is duplicate code from: https://github.com/uzh-rpg/e2calib/blob/cea480c80822f0b91a851609166d6c71cf341e5f/python/conversion/ros.py#L8

We should have a separate file and import the class

File: python/data/provider.py (hunk 2 above)
Comment (jhidalgocarrio) [reply python/data/provider.py:127]: because two events could have the same timestamp, correct?

File: python/conversion/pocolog.py (hunk 3 above)
Comment (jhidalgocarrio) [reply python/conversion/pocolog.py:12]: This is assuming ros EventAccumulator and pocolog EventAccumulator are the same. It is true now there are the same but it might change in the future. right?

File: python/pocolog_convert.py (hunk 1 above)
Comment (jhidalgocarrio) [reply python/pocolog_convert.py:1]: yes, sorry I will delete this file. Historical reasons. I thought I already deleted it.

File: python/conversion/pocolog.py (hunk 3 above)
Comment (magehrig) [reply python/conversion/pocolog.py:12]: Atm they are the same and we can reduce code duplication. We could still change back later if needed

File: python/data/provider.py (hunk 2 above)
Comment (magehrig) [reply python/data/provider.py:127]: ok, I think it does not matter and it's fine like this

Comment (jhidalgocarrio) [reply python/data/provider.py:127]: all right. thanks for your comment

File: python/conversion/pocolog.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        self.x = list()
+        self.y = list()
+        self.p = list()
+        self.t = list()
+
+    def add_event(self, event):
+        self.x.append(event.x)
+        self.y.append(event.y)
+        self.p.append(int(event.polarity))
+        self.t.append(event.ts.to_microseconds())
```
Comment (jhidalgocarrio) [inline python/conversion/pocolog.py:23]: @magehrig Actually. This part of the code is different in EventAcumulator for ros and for pocolog format. Pocolog timestamp does not have to_nsec method and changing it is not in my hands.

I am open to other suggestions but it seems we need to have two separated EventAccumulator classes.

File: python/conversion/pocolog.py (hunk 3 above)
Comment (jhidalgocarrio) [reply python/conversion/pocolog.py:12]: I just tried to merge them but they're actually different. Please have a look at my comment.

File: python/conversion/pocolog.py (hunk 4 above)
Comment (magehrig) [reply python/conversion/pocolog.py:23]: You can create a Base class with all but the non-shareable functionality
